react-native-purchases-ui/android/src/main/java/com/revenuecat/purchases/react/ui/PaywallViewManager.kt

```diff
@@ -22,4 +24,18 @@ internal class PaywallViewManager : SimpleViewManager<PaywallView>() {
     override fun createShadowNodeInstance(): PaywallViewShadowNode {
         return PaywallViewShadowNode()
     }
+
+    @ReactProp(name = "options")
+    fun setOptions(view: PaywallView, options: ReadableMap?) {
```

Comment:
I wonder if we could avoid this repetition somehow... Maybe we could have some kind of interface to at least parse the options?

Reply:
I like it

Reply:
what do you think @tonidero https://github.com/RevenueCat/react-native-purchases/pull/850/commits/1bc9eebec9cc9835b80785c5e73bdaa7db99381d

Reply:
Love it! 👍